assets/search.js

```diff
@@ -1,619 +1,666 @@
-        let iEl = document.createElement("i");
-        iEl.className = "material-icons bucketlist-add";
-        iEl.textContent = "favorite_border";
-        buttonEl.appendChild(iEl);
-
-        // create div for content
-        divEl = document.createElement("div");
-        cardEl.appendChild(divEl);
-        divEl.className = "card-content";
-
-        // place name
-        let spanEl = document.createElement("span");
-        spanEl.className = "card-title activator";
-        spanEl.textContent = place.name;
-        divEl.appendChild(spanEl);
-
-        // place address
-        pEl = document.createElement("p");
-        divEl.appendChild(pEl);
-        pEl.textContent = place.formatted_address;
-
-        // star rating
-        pEl = document.createElement("p");
-        divEl.appendChild(pEl);
-        var rating = Math.round(place.rating); //round rating to nearest integer
-        if (rating !== 0) {
-            for (var i = 1; i <= rating; i++) {
-                pEl.textContent += "✭";
-            }
-        } else {
-            pEl.textContent = "No rating";
+  console.log("displaying places");
+  let placeArray = [];
+  let places = JSON.parse(localStorage.getItem("gTextData")).eateries;
+  console.log(places);
+  let displayEl = document.getElementById("google-results");
+  let index = 0;
+  let randomNumCount = 0;
+
+  // array for random photos in case an eatery lacks photos
+  var eateryPhotos = [];
+  eateryPhotos = [
+    "./assets/eatery-photos/eatery-1.jpg",
+    "./assets/eatery-photos/eatery-2.jpg",
+    "./assets/eatery-photos/eatery-3.jpg",
+    "./assets/eatery-photos/eatery-4.jpg",
+    "./assets/eatery-photos/eatery-5.jpg",
+    "./assets/eatery-photos/eatery-6.jpg",
+    "./assets/eatery-photos/eatery-7.jpg",
+    "./assets/eatery-photos/eatery-8.jpg",
+    "./assets/eatery-photos/eatery-9.jpg",
+    "./assets/eatery-photos/eatery-10.jpg",
+  ];
+
+  var randomNums = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+  shuffle(randomNums);
+
+  places.forEach((place) => {
+    //create card
+    //add logic to not display closed businesses, but keep increasing the index of the array goes up
+    if (place.business_status !== "OPERATIONAL") {
+      console.log("no card created for closed business");
+    } else {
+      let cardEl = document.createElement("div");
+      cardEl.className = "card search-card";
+      cardEl.id = `card-${index}`;
+      displayEl.appendChild(cardEl);
+      //create array for cardEl.id and corresponding place_id
+      placeArray.push(place.place_id);
+      index++;
+
+      // div for image
+      let divEl = document.createElement("div");
+      divEl.className = "card-image";
+      cardEl.appendChild(divEl);
+
+      // image
+      let imgEl = document.createElement("img");
+      imgEl.className = "activator";
+      // grab image from eateryPhotos array, using shuffled randomNums array
+      if (randomNumCount != randomNums.length) {
+        let r = randomNums[randomNumCount];
```

Comment:
you don't need to declare a variable for your randomnums[randomNumCount] here since you are just using it on the next line
you can pop that directly into line 336 as
`imgEl.setAttribute("src", eateryPhotos[randomNums[randomNumCount]])`
(or index, if you switch to it)